docs/docsite/rst/reference_appendices/module_utils.rst

```diff
@@ -5,65 +5,25 @@
 Ansible Reference: Module Utilities
 ***************************************************************
 
-This page documents the available utilities called with ``ansible.module_utils.util_name``.
+This page documents utilities intended to be helpful when writing
+Ansible modules in Python.
 
-Generic
---------
-.. glossary::
-
-    .. _ansible.module_utils.debug
-    debug
-    .. _ansible.module_utils.url
-    url
-    .. _ansible.module_utils.log
-    log
-    .. _ansible.module_utils.no_log
-    no_log
-    .. _Ansible.get_bin_path
-    Ansible.get_bin_path
-
-
-.. _AnsibleModule:
-.. _ansible.module_utils.basic.AnsibleModule:
 
 AnsibleModule
 --------------
-.. glossary::
 
-    ansible.module_utils.basic.AnsibleModule
-      Utilities in module_utils.basic.AnsibleModule apply to all module types
+This functionality is available via ``from ansible.module_utils.basic import AnsibleModule``.
 
-    .. _AnsibleModule.debug:
-    AnsibleModule.debug
-    .. _AnsibleModule._debug:
-    AnsibleModule._debug
-    .. _AnsibleModule._diff:
-    AnsibleModule._diff
-    .. _AnsibleModule.log:
-    AnsibleModule.log
-    .. _AnsibleModule.no_log:
-    AnsibleModule.no_log
-    .. _AnsibleModule.params:
-    AnsibleModule.params
-    .. _AnsibleModule.run_command:
-    AnsibleModule.run_command
-    .. _ansible.module_utils.basic.AnsibleModule._selinux_special_fs:
-    ansible.module_utils.basic.AnsibleModule._selinux_special_fs
-      (formerly ansible.module_utils.basic.SELINUX_SPECIAL_FS)
+.. autoclass:: ansible.module_utils.basic.AnsibleModule
+   :members:
 
 
-.. _ansible.module_utils.basic:
-
 Basic
 ------
-.. glossary::
 
-    ansible.module_utils.basic
-      Utilities in module_utils.basic apply to all module types.
+This functionality is available via ``import ansible.module_utils.basic``:
+
+.. automodule:: ansible.module_utils.basic
+   :members:
```

Comment:
What does `members` limit this to?

Reply:
"Limit" does not describe it very well... Just saying `automodule::` just gives the doc string of the module itself. I considered it useful to also have the documentation of the module's members, so I added `:members:`. The [sphinx doc](http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html) explains this.